### DISCUSSION TITLE
Why am I getting an error when specifying a revset with `jj log -r`?

### DISCUSSION THREAD

**Discussion by mybuddymichael** [jj-vcs/jj#1943] [Q&A] [answered]
I'm working through the tutorial and attempting to run `jj log -r master::@`. When I do I get this error:

```
Error: Failed to parse revset:  --> 1:8
  |
1 | master::@
  |        ^---
  |
  = expected EOI, union_op, intersection_op, difference_op, compat_add_op, compat_sub_op, or primary
```

Am I doing something wrong?

**Comment by arxanas** [comment]
I think @martinvonz just changed between `:` and `::` (I forget which is old and which is new), and the documentation doesn't match the version of jj you're using (did you install from source or from your package manager?).

**Reply by martinvonz** [reply, marked as answer]
Yes, that's correct. Sorry, I should have delayed updating the documentation. Use `:` for now (or install an unreleased version from source).

**Reply by mybuddymichael** [reply]
I installed from Homebrew. Thanks!

**Reply by martinvonz** [reply]
FYI, I've sent PR #1945 to revert our documentation to the `:` operator that's available in our latest released version (0.8.0).
